UX-Guide-Metadata/draft/principles/index.html

```diff
@@ -360,7 +360,7 @@ <h4>Examples</h4>
 							<li>Readable in read aloud and braille.</li>
 							<li>May not be fully readable in read aloud and braille.</li>
 							<li>Not fully readable in read aloud and braille.</li>
-<li>Not known ifReadable in read aloud and braille</li>
+<li>Not known if Readable in read aloud and braille</li>
```

Comment:
Readable shouldn't be capitalized in the middle of the sentence here.
```suggestion
<li>Not known if readable in read aloud and braille</li>
```